src/components/WalletBalanceList/index.tsx

```diff
@@ -69,30 +78,60 @@ export const WalletBalanceList: React.SFC<WalletBalanceListProps> = ({
                       {asBalance(b)} {b.asset.name}
                     </td>
                     <td className="_action">
-                      <Dropdown
-                        overlay={
-                          <div className="asset-menu">
-                            <div>
-                              <Link to={ROUTE_TRANSFER_TO(wallet.id)}>
-                                Deposit
-                              </Link>
-                            </div>
-                            <div>
-                              <Link
-                                to={ROUTE_TRANSFER_FROM(wallet.id, b.assetId)}
-                              >
-                                Withdraw
-                              </Link>
-                            </div>
-                          </div>
-                        }
-                        trigger={['click']}
-                        placement="bottomCenter"
-                      >
-                        <button type="button" className="btn btn--icon">
-                          <i className="icon icon--actions" />
-                        </button>
-                      </Dropdown>
+                      {(b.asset.isBankDepositEnabled || !wallet.isTrading) && (
+                        <Dropdown trigger="click">
+                          <DropdownControl>
+                            <button type="button" className="btn btn--icon">
+                              <i className="icon icon--actions" />
+                            </button>
+                          </DropdownControl>
+                          <DropdownContainer>
+                            <DropdownList className="asset-menu">
+                              {wallet.isTrading
+                                ? [
```

Comment:
React 16 introduces pretty cool `Fragment` syntax, so we can use it.
But i'm not sure we're running React 16, so upgrading to it seems a good idea.

Not demanding for this PR